docs/extras/modules/data_connection/vectorstores/integrations/meilisearch.ipynb

```diff
@@ -68,87 +91,60 @@
     "import os\n",
     "import getpass\n",
     "\n",
+    "# Prompt for Meilisearch credentials\n",
+    "os.environ[\"MEILI_HTTP_ADDR\"] = getpass.getpass(\"Meilisearch HTTP address and port:\")\n",
+    "os.environ[\"MEILI_MASTER_KEY\"] = getpass.getpass(\"Meilisearch API Key:\")\n",
+    "\n",
+    "# Prompt for other third-party credentials\n",
     "os.environ[\"OPENAI_API_KEY\"] = getpass.getpass(\"OpenAI API Key:\")"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "If you don't want to have to send your client or Meilisearch instance address, you can directly configure the [environment variables](https://www.meilisearch.com/docs/learn/configuration/instance_options#environment-variables)."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "os.environ[\"MEILI_HTTP_ADDR\"] = getpass.getpass(\"Meilisearch HTTP address and port:\")"
+    "### Adding text and embeddings\n",
+    "\n",
+    "This example adds text to the Meilisearch vector store along with the associated embeddings."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
    "outputs": [],
-   "source": [
-    "os.environ[\"MEILI_MASTER_KEY\"] = getpass.getpass(\"Meilisearch API Key:\")"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "## Add texts with user-friendly interface"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 5,
-   "metadata": {},
-   "outputs": [],
    "source": [
     "from langchain.vectorstores import Meilisearch\n",
     "from langchain.embeddings.openai import OpenAIEmbeddings\n",
     "from langchain.text_splitter import CharacterTextSplitter\n",
     "\n",
-    "embeddings = OpenAIEmbeddings()"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 6,
-   "metadata": {},
-   "outputs": [],
-   "source": [
+    "# OpenAI API key is loaded from environment\n",
+    "embeddings = OpenAIEmbeddings()\n",
+    "\n",
+    "# Read the file content\n",
     "with open(\"../../../state_of_the_union.txt\") as f:\n",
     "    state_of_the_union = f.read()\n",
     "text_splitter = CharacterTextSplitter(chunk_size=1000, chunk_overlap=0)\n",
-    "texts = text_splitter.split_text(state_of_the_union)"
+    "texts = text_splitter.split_text(state_of_the_union)\n",
+    "\n",
+    "# Use Meilisearch vector store to store texts & associated embeddings\n",
+    "vector_store = Meilisearch.from_texts(texts=texts, embedding=embeddings)"
```

Comment:
I've used a different name for these two examples to differentiate this from Meilisearch's direct initialization. However, the name may not speak for itself. However, I find it confusing to call it `vector_store` and then use `vectorstore`, so maybe it's better to use the same name everywhere. WDYT?

Reply:
Yes, even both `docsearch` in similarity search